packages/dashboard/src/helpers/updateChildFilters.ts

```diff
@@ -0,0 +1,44 @@
+import { SharedFilter } from '../types/SharedFilter'
+import _ from 'lodash'
+
+export const updateChildFilters = (newSharedFilters: SharedFilter[], data: Record<string, any>): SharedFilter[] => {
```

Comment:
Can you add logic to this function to only run if the filter is a data filter? api filters handle parent relationships differently

Reply:
Added logic to filter out all data filters